### PR TITLE
feat: Adds information about the MCA setup

### PR DESCRIPTION
**Pull request by tfelix**

### CODE REVIEW

File: docs/meshstack.how-to.integrate-meshplatform-azure-manually.md [hunk 1]
```diff
@@ -257,7 +257,30 @@ In contrast to other provisioning methods, EA provisioning will not retain a def
 
 > You should never grant subscription owner roles to the meshStack replicator SPN.
 
-### Option 2: Using pre-provisioned Subscriptions
+### Using a Customer Agreement
```
Comment (felixzieger) [inline docs/meshstack.how-to.integrate-meshplatform-azure-manually.md:260]:
```suggestion
### Use a Customer Agreement
```

File: docs/meshstack.how-to.integrate-meshplatform-azure-manually.md [hunk 2]
```diff
@@ -257,7 +257,30 @@ In contrast to other provisioning methods, EA provisioning will not retain a def
...
+
+The requirements for the principal on the target tenant is identically with the one from the Enterprise Agreement provisioning. Please follow the section [Set up the Replication Service Principal](#set-up-the-replication-service-principal).
+
+#### meshPanel Configuration
+
+When creating a new platform configuration in the meshPanels admin section, choose Customer Agreement and fill in the form with the data you just noted down. The `Principal Client Secret`, `Principal Object ID` and `Billing Account Principal Client ID` refer to the data of the service principal from the billing/source tenant AAD.
+
+The `Destination AAD ID` must be an ID. Please don't use the domain name variant of it e.g. `*.onmicrosoft.com` but use its UUID form. The principal data in the **Access Configuration** part refers to the target tenant AAD service principal.
+
+### Using pre-provisioned Subscriptions
```
Comment (felixzieger) [inline docs/meshstack.how-to.integrate-meshplatform-azure-manually.md:283]:
```suggestion
### Use pre-provisioned Subscriptions
```

File: docs/meshstack.how-to.integrate-meshplatform-azure-manually.md [hunk 3]
```diff
@@ -257,7 +257,30 @@ In contrast to other provisioning methods, EA provisioning will not retain a def
...
+
+> Currently meshStack only supports the MCA tenant creation across different AAD tenants. See [Microsofts documentation](https://learn.microsoft.com/en-us/azure/cost-management-billing/manage/programmatically-create-subscription-microsoft-customer-agreement-across-tenants) about it.
+
+#### Create Source Tenant Principal
+
+1. On the tenant that contains your billing account create a new Enterprise Application.
+2. Add credentials to it and note down the `Directory (tenant) ID`, `Application (client) ID`, `Object ID` and the `secret` just generated by you.
+3. Select the billing profile or invoice section under which you want to create the subscriptions. You can follow the [documentation](https://learn.microsoft.com/en-us/azure/cost-management-billing/manage/programmatically-create-subscription-microsoft-customer-agreement?tabs=rest#find-billing-profiles--invoice-sections-to-create-subscriptions) from Microsoft. You need to write down the ID that lookes something like this `/providers/Microsoft.Billing/billingAccounts/5e98e158-xxxx-xxxx-xxxx-xxxxxxxxxxxx:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx_xxxx-xx-xx/billingProfiles/AW4F-xxxx-xxx-xxx/invoiceSections/SH3V-xxxx-xxx-xxx`.
```
Comment (felixzieger) [inline docs/meshstack.how-to.integrate-meshplatform-azure-manually.md:270]:
```suggestion
3. Select the billing profile or invoice section under which you want to create the subscriptions by following the [documentation](https://learn.microsoft.com/en-us/azure/cost-management-billing/manage/programmatically-create-subscription-microsoft-customer-agreement?tabs=rest#find-billing-profiles--invoice-sections-to-create-subscriptions) from Microsoft. Write down the ID that looks something like this `/providers/Microsoft.Billing/billingAccounts/5e98e158-xxxx-xxxx-xxxx-xxxxxxxxxxxx:xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx_xxxx-xx-xx/billingProfiles/AW4F-xxxx-xxx-xxx/invoiceSections/SH3V-xxxx-xxx-xxx`.
```

**Review by felixzieger — APPROVED** [review]
Very cool!

File: docs/meshstack.how-to.integrate-meshplatform-azure-manually.md (hunk 2 above)
Comment (tfelix) [reply docs/meshstack.how-to.integrate-meshplatform-azure-manually.md:283]: fixed

**Pull request merged**